### ISSUE TITLE
efficientnet for regression 

### ISSUE THREAD

**Issue by segalinc** [lukemelas/EfficientNet-PyTorch#261]
it's not super clear if the output of the model is from the fc layer  or the one coming from the swish module that goes through a sigmoid. 
I have labels ranging (-1,1) so I would need to know that in order to either scale them or take off the last layer

**Comment by lukemelas** [comment]
[I'm aware you've probably already figured this out -- this is primarily for everyone who sees this in the future]

If you're using the model in a standard way, it's the output of the fc layer. There should be no need to remove the fc layer, just make `num_classes=1` and use a loss function for regression.
